File: Sem 4/Flutter/Lab 4/count_odd_even.dart
import 'dart:io';

void main() {
  List arr = [];
  int odd = 0, even = 0, n;

  stdout.write("Enter n : ");
  n = int.parse(stdin.readLineSync()!);

  for (int i = 0; i < n; i++) {
    stdout.write("Enter value : ");
    arr.add(int.parse(stdin.readLineSync()!));
  }
  for (int i in arr) {
    if (i % 2 == 0) {
      even++;
    } else {
      odd++;
    }
  }
  print("odd = ${odd}\n even = ${even}");
  return;
}

File: Sem 1/C-lab/For exam/count_odd_even.c
#include<stdio.h>

int main(){
    int n, odd=0, even=0;
    scanf("%d",&n);

    int arr[n];
    for (int i = 0; i < n; i++)
    {
        printf("Enter Array : ");
        scanf("%d",&arr[i]);
        if (arr[i] % 2 == 0) { even++; }
        else {odd++;}
    }
    printf("odd = %d  even = %d", odd, even);
}